docs/machine-learning/tutorials/sales-anomaly-detection.md

```diff
@@ -74,48 +74,48 @@ Add a new class to your project:
 
 2. In the **Add New Item dialog box**, select **Class** and change the **Name** field to *ProductSalesData.cs*. Then, select the **Add** button.
 
-The *ProductSalesData.cs* file opens in the code editor. Add the following `using` statement to the top of *ProductSalesData.cs*:
-
-```csharp
-using Microsoft.ML.Data;
-```
-
-Remove the existing class definition and add the following code, which has two classes `ProductSalesData` and `ProductSalesPrediction`, to the *ProductSalesData.cs* file:
+   The *ProductSalesData.cs* file opens in the code editor.
 
-[!code-csharp[DeclareTypes](~/samples/machine-learning/tutorials/ProductSalesAnomalyDetection/ProductSalesData.cs#DeclareTypes "Declare data record types")]
+3. Add the following `using` statement to the top of *ProductSalesData.cs*:
 
-`ProductSalesData` specifies an input data class. The [LoadColumn](xref:Microsoft.ML.Data.LoadColumnAttribute.%23ctor%28System.Int32%29) attribute specifies which columns (by column index) in the dataset should be loaded. 
+   ```csharp
+   using Microsoft.ML.Data;
+   ```
 
-Add the following additional `using` statements to the top of the *Program.cs* file:
+4. Remove the existing class definition and add the following code, which has two classes `ProductSalesData` and `ProductSalesPrediction`, to the *ProductSalesData.cs* file:
 
-[!code-csharp[AddUsings](~/samples/machine-learning/tutorials/ProductSalesAnomalyDetection/Program.cs#AddUsings "Add necessary usings")]
+    [!code-csharp[DeclareTypes](~/samples/machine-learning/tutorials/ProductSalesAnomalyDetection/ProductSalesData.cs#DeclareTypes "Declare data record types")]
 
-You need to create two global fields to hold the recently downloaded dataset file path and the saved model file path:
+    `ProductSalesData` specifies an input data class. The [LoadColumn](xref:Microsoft.ML.Data.LoadColumnAttribute.%23ctor%28System.Int32%29) attribute specifies which columns (by column index) in the dataset should be loaded.
```

Comment:
Suggestion:

Since input data class is explained, also explain ProductSalesPrediction.

Reply:
Good catch. Will do